### ISSUE TITLE
let {esp,ah}=... imply phase2=...

### ISSUE THREAD

**Issue by cagney** [libreswan/libreswan#712]
Having to specify both:
  ah=sha1
  phase2=ah
is redundant (and annoying when editing lots of tests)

**Comment by letoams** [comment]
Reminds me, we should add USE_AH=false

Sent using a virtual keyboard on a phone

**Comment by cagney** [comment]
> When phase2=is absent the connection is considered never-negotiate. A never negotiate connection warns and ignores esp=.

this is wrong; the thing making a connection never-negotiate is `type=...`
